django/utils/translation/trans_real.py

```diff
@@ -509,6 +509,7 @@ def get_language_from_request(request, check_path=False):
         return settings.LANGUAGE_CODE
 
 
+@functools.lru_cache(maxsize=128)
```

Comment:
The ticket (and probably a code comment) should also explain how `maxsize=128` was chosen.

Reply:
At least it isn't `None`, which would have been a security vulnerability ;-)

Reply:
Why not 1000 like several other functions in this module?

Reply:
No particular reason, other than I had seen 128 used before. I guess 1,000 is just as good.